src/AbstractDifferentiation.jl

```diff
@@ -246,7 +253,11 @@ function value_and_pullback_function(
                 value = primalvalue(lowest(ab), vs, f, xs)
                 primalcalled = true
             end
-            return vs
+            if vs isa Tuple
```

Comment:
why are we doing this?

Reply:
that one was an old artefact. I removed it.